research/a3c_blogpost/a3c_cartpole.py

```diff
@@ -14,7 +14,7 @@
 from tensorflow.python import keras
 from tensorflow.python.keras import layers
 
-tf.enable_eager_execution()
+tf.compat.v1.enable_eager_execution()
```

Comment:
with tf 2.x, I think this line is not needed. Could you test it out?

Reply:
Test finish.
'''
Traceback (most recent call last):
  File "a3c_cartpole.py", line 17, in <module>
    tf.enable_eager_execution()
AttributeError: module 'tensorflow' has no attribute 'enable_eager_execution'
'''

Reply:
Yes, I can remove this line, and it work perfect.This line is not needed.